modules/wallet/money.go

```diff
@@ -13,17 +13,25 @@ type sortedOutputs struct {
 	outputs []types.SiacoinOutput
 }
 
+// managedDustThreshold returns the quantity below which a Currency is considered to be Dust.
+func (w *Wallet) managedDustThreshold() types.Currency {
+	minFee, _ := w.tpool.FeeEstimation()
+	return minFee.Mul64(3)
+}
+
 // ConfirmedBalance returns the balance of the wallet according to all of the
 // confirmed transactions.
 func (w *Wallet) ConfirmedBalance() (siacoinBalance types.Currency, siafundBalance types.Currency, siafundClaimBalance types.Currency) {
+	dustThreshold := w.managedDustThreshold()
```

Comment:
probably worth adding a quick comment to each of these lone `dustThreshold :=` calls